hack/verify-typecheck-providerless.sh

```diff
@@ -27,3 +27,11 @@ cd "${KUBE_ROOT}"
 # verify the providerless build
 # https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/1179-building-without-in-tree-providers/README.md
 hack/verify-typecheck.sh --skip-test --tags=providerless --ignore-dirs=test
+
+# verify using go list
+if _out="$(go list -mod=readonly -tags "providerless" -e -json  k8s.io/kubernetes/cmd/kubelet/... \
```

Comment:
only kubelet?

Reply:
Let's see if this holds up before adding more in 1.22 :)

Reply:
Ideally I want to add a restrict regex in the typecheck itself